chart/templates/workers/worker-deployment.yaml

```diff
@@ -256,6 +257,10 @@ spec:
             - name: worker-logs
               containerPort: {{ .Values.ports.workerLogs }}
           volumeMounts:
+            {{- if and (semverCompare ">=2.8.0" .Values.airflowVersion) .Values.workers.kerberosSidecar.enabled (semverCompare "<1.28" $kubeVersion) }}
```

Comment:
Looks like 2.8 will change now, no? I see the RC out for 2.8 already. 
cc @potiuk